internal/consensus/byzantine_test.go

```diff
@@ -540,8 +541,8 @@ func sendProposalAndParts(
 
 	// votes
 	cs.mtx.Lock()
-	prevote, _ := cs.signVote(types.PrevoteType, blockHash, parts.Header(), nil)
-	precommit, _ := cs.signVote(types.PrecommitType, blockHash, parts.Header(), nil)
+	prevote, _ := cs.signVote(types.PrevoteType, blockHash, parts.Header(), block)
+	precommit, _ := cs.signVote(types.PrecommitType, blockHash, parts.Header(), block)
```

Comment:
Otherwise, vote extensions don't work as expected.

Reply:
should this be a different PR?

Reply:
I think you can leave the `prevote` one (line 544) to `nil` and it'll still work.
As for the `precommit`... oh my! I looked into it and I think you're right, there's no other way, as the production code is calling `ExtendVote` with the block (and checking that the block we give makes sense).

@lasarojc I think it has to be this PR, otherwise the tests won't pass. @cason unless we decide to run **just this test case** with vote extensions disabled (but I'm also OK with the current solution)

Reply:
> I think you can leave the prevote one (line 544) to nil and it'll still work.

Indeed, reverting.